apps/nativeapp/app/screens/CreatePolygon/CreatePolygon.tsx

```
@@ -27,27 +27,29 @@ import {
   CustomButton,
   FloatingInput,
 } from '../../components';
-import Map from './mapMarking/map';
+import Map from './MapMarking/Map';
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)